eng/build.yml

```diff
@@ -91,10 +91,10 @@ jobs:
       pool:
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
           name: $(DncEngPublicBuildPool)
-          demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+          demands: ImageOverride -equals Build.Ubuntu.2204.Amd64.Open
```

Comment:
Using an old Ubuntu build caused a warning in CI. This was also seen in arcade, and they simply bumped the image version here: https://github.com/dotnet/arcade/pull/14286